services/QuillLMS/app/uploaders/admin_report_csv_uploader.rb

```diff
@@ -1,22 +1,23 @@
 # frozen_string_literal: true
 
-class AdminReportCsvUploader < CarrierWave::Uploader::Base
-  attr_reader :admin_id
+class AdminReportCsvUploader < ApplicationUploader
+  FILENAME_PREFIX = 'ADMIN_REPORT_'
 
-  # Max value is 604800, 7 days
   # See: https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-query-string-auth.html
-  FILE_EXPIRATION = 604800
-
-  fog_directory ENV.fetch('ADMIN_REPORT_FOG_DIRECTORY', 'admin-report-fog-directory-staging')
-  fog_authenticated_url_expiration FILE_EXPIRATION
+  fog_authenticated_url_expiration 7.days.to_i
+  fog_directory ADMIN_REPORT_FOG_DIRECTORY
 
-  FILENAME_PREFIX = 'ADMIN_REPORT_'
+  attr_reader :admin_id
 
   def initialize(model = nil, mounted_as = nil, admin_id:)
     @admin_id = admin_id
     super(model, mounted_as)
   end
 
+  def filename
+    "ADMIN_REPORT_#{@admin_id}_#{date}_#{generate_token}.csv"
```

Comment:
Should this reference `FILENAME_PREFIX` rather than being the string literal?

Reply:
Yes.  It should.  Good catch.